index.js

```diff
@@ -191,12 +213,16 @@ FocusGroup.prototype._findIndexOfNode = function(searchNode) {
 };
 
 FocusGroup.prototype._getActiveElementIndex = function() {
-  return this._findIndexOfNode(document.activeElement);
+  return this._rootNode !== document
+    ? this._activeIndex
+    : this._findIndexOfNode(document.activeElement);
 };
 
 FocusGroup.prototype.focusNodeAtIndex = function(index) {
   var member = this._members[index];
   if (member) focusNode(member.node);
```

Comment:
What if your fake focus group contains anchors or buttons, though? I think it's still conceivable that you'd want to use this feature, right?